### PR TITLE
Add a few different tools

### PR DESCRIPTION
**Pull request by adds68**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ Infrastructure as Code allows applications to be deployed reliably to a consiste
 - [Hadolint](https://github.com/hadolint/hadolint) - _Hadolint_ - Checks a Dockerfile against known rules and validates inline bash code in RUN statements.
 - [Snyk Container](https://snyk.io/product/container-vulnerability-management/) - _Snyk_ - Scan Docker and Kubernetes applications for security vulnerabilities during CI/CD or via continuous monitoring.
 - [Trivy](https://github.com/aquasecurity/trivy) - _Aqua Security_ - Simple and comprehensive vulnerability scanner for containers.
+- [Docker-Bench-Security](https://github.com/docker/docker-bench-security) - _Docker_ - The Docker Bench for Security is a script that checks for dozens of common best-practices around deploying Docker containers in production
```
Comment (iamscottcab) [inline readme.md:142]: Can this be moved up so it's in alpahbetical order?

File: readme.md [hunk 2]
```diff
@@ -187,6 +191,7 @@ The software we write needs to use secrets (passwords, API keys, certificates, d
 - [HashiCorp Vault](https://www.vaultproject.io/) - _HashiCorp_ - Securely store secrets via UI, CLI or HTTP API.
 - [Pinterest Knox](https://github.com/pinterest/knox) - _Pinterest_ - Securely store, rotate and audit secrets.
 - [Secrets Operations (SOPS)](https://github.com/mozilla/sops) - _Mozilla_ - Encrypt keys stored within YAML, JSON, ENV, INI and BINARY files.
+- [Detect Secrets](https://github.com/Yelp/detect-secrets) - _Yelp_ - detect-secrets is an aptly named module for (surprise, surprise) detecting secrets within a code base.
```
Comment (iamscottcab) [inline readme.md:194]: Can this be moved up so it's in alpahbetical order?

File: readme.md [hunk 3]
```diff
@@ -202,6 +207,7 @@ Static Analysis Security Testing (SAST) tools scan software for vulnerabilities
 - [RIPS](https://www.ripstech.com/) - _RIPS Technologies_ - Automated static analysis for PHP, Java and Node.js projects.
 - [SonarLint](https://www.sonarlint.org/) - _SonarSource_ - An IDE plugin that highlights potential security security issues, code quality issues and bugs. 
 - [SonarQube](https://www.sonarqube.org/) - _SonarSource_ - Scan code for security and quality issues with support for a wide variety of languages.
+- [SemGrep](https://semgrep.dev/) - _r2c_ - Semgrep is a fast, open-source, static analysis tool that finds bugs and enforces code standards at editor, commit, and CI time.
```
Comment (iamscottcab) [inline readme.md:210]: Can this be moved up so it's in alpahbetical order?

**Review by iamscottcab — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (adds68) [reply readme.md:142]: sure!

**Comment by adds68** [comment]
@iamscottcab Changes are ready for review

**Comment by iamscottcab** [comment]
> @iamscottcab Changes are ready for review

Thanks I'll merge these now 👍

**Review by iamscottcab — APPROVED** [review]

**Pull request merged**